src/Button.tsx

```diff
@@ -0,0 +1,34 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at https://mozilla.org/MPL/2.0/. */
+
+import './Button.css';
+
+interface Props {
+  color: string;
+  size: string;
+  textToDisplay: string;
+  clickFunction: () => void;
+  disabledFunction?: () => boolean;
+}
+
+const setButtonName = (color: string, size: string) => {
+  return color + ' ' + size;
+};
```

Comment:
function is called `setButtonName`, but doesn't actually set anything - maybe `constructButtonClass` or something else more clear

Reply:
For sure. I have changed the function name to be `constructButtonClass` now.